.github/workflows/bun.yml

```diff
@@ -174,6 +174,13 @@ jobs:
         run: |
           mkdir -p $BUN_DEPS_OUT_DIR
           make vendor-without-check
+      - name: Cache C++
+        if: matrix.compile_obj
+        id: cache-cpp
+        uses: actions/cache@v3
+        with:
+          path: ${{ runner.temp }}/bun-cpp-obj
+          key: ${{ runner.os }}-${{ matrix.cpu }}-${{ matrix.arch }}-cpp
```

Comment:
might also want to do `hashFiles()` here like in [this example](https://github.com/actions/cache#creating-a-cache-key)